test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -312,18 +312,16 @@ const std::string UcxContext::sockaddr_str(const struct sockaddr* saddr,
     case AF_INET:
         inet_ntop(AF_INET, &((struct sockaddr_in*)&addr)->sin_addr,
                   buf, sizeof(buf));
-        port = ntohs(((struct sockaddr_in*)&addr)->sin_port);
         break;
     case AF_INET6:
         inet_ntop(AF_INET6, &((struct sockaddr_in6*)&addr)->sin6_addr,
                   buf, sizeof(buf));
-        port = ntohs(((struct sockaddr_in6*)&addr)->sin6_port);
         break;
     default:
         return "<invalid address>";
     }
-
-    snprintf(buf + strlen(buf), sizeof(buf) - strlen(buf), ":%d", port);
+    ucs_sockaddr_get_port(saddr, &port);
```

Comment:
1. why needed to change?
2. pls don't use UCS functions from io_demo

Reply:
With gcc4.4.7 there were below "strict-aliasing" build errors on line298 & ln303 ucx_wrapper.cc in ucx1.11.2.
so I made a new commit not calling ucs.

test/apps/iodemo/ucx_wrapper.cc
 298         port = ntohs(((struct sockaddr_in*)&addr)->sin_port);

ucx_wrapper.cc: In static member function ‘static const std::string UcxContext::sockaddr_str(const sockaddr*, size_t)’:
ucx_wrapper.cc:298: error: dereferencing pointer ‘addr.108’ does break strict-aliasing rules
ucx_wrapper.cc:298: note: initialized from here
ucx_wrapper.cc:303: error: dereferencing pointer ‘addr.110’ does break strict-aliasing rules
ucx_wrapper.cc:303: note: initialized from here